### DISCUSSION TITLE
What are the parameters for 'createSoftBodyAnchor'?

### DISCUSSION THREAD

**Discussion by aprath1** [bulletphysics/bullet3#4088] [General]
Hi I am working on understanding soft bodies in pybullet. I saw a functionality to create soft body anchor in the example file - https://github.com/bulletphysics/bullet3/blob/master/examples/pybullet/examples/deformable_anchor.py

But there was no documentation about the parameters of createSoftBodyAnchor in the PyBullet Quickstart Guide. 
->For example below is line 23 in deformable_anchor.py:
    p.createSoftBodyAnchor(clothId  ,24,-1,-1)
    What does 24, -1 and -1 stand for ?
->below is line 25 in d
    p.createSoftBodyAnchor(clothId ,15,boxId,-1, [0.5,-0.5,0])
    Here what does 15, [0.5, -0.5, 0] indicate?
Is there a repo where all the functions mentioned in the [pybullet quick start guide](https://docs.google.com/document/d/10sXEhzFRSnvFcl3XxNGhnD4N2SedqwdAvK3dsihxVUA/edit#heading=h.2ye70wns7io3) is defined ?

**Comment by aprath1** [comment]
Guess the line 686 in [Bullet C API](https://github.com/bulletphysics/bullet3/blob/master/examples/SharedMemory/PhysicsClientC_API.h) gives an insight to this. 
But it would be great if any one can share where are all the python wrappers or binding functions in the  [pybullet quick start guide](https://docs.google.com/document/d/10sXEhzFRSnvFcl3XxNGhnD4N2SedqwdAvK3dsihxVUA/edit#heading=h.2ye70wns7io3) is defined...?

**Comment by jmz3** [comment]
Hopefully they can get a python API someday...
